docs/en/index.rst

```diff
@@ -1,8 +1,55 @@
 Doctrine ORM Module for Laminas
 ===============================
 
-This module integrates `Doctrine ORM <https://www.doctrine-project.org/projects/orm.html>`_
-with `Laminas <https://getlaminas.org/>`_.
+The DoctrineORMModule leverages `DoctrineModule <https://www.doctrine-project.org/projects/doctrine-module/en/current/index.html>`__
+and integrates `Doctrine ORM <https://www.doctrine-project.org/projects/doctrine-orm/en/current/index.html>`__
+with `Laminas <https://getlaminas.org/>`__ quickly and easily. The following features are intended
+to work out of the box:
+
+  - Doctrine ORM support
+  - Multiple ORM entity managers
+  - Multiple DBAL connections
+  - Reuse existing PDO connections in DBAL connection
+
+Installation
+------------
+
+Run the following to install this library using `Composer <https://getcomposer.org/>`__:
+
+.. code:: bash
+
+   $ composer require doctrine/doctrine-orm-module
+
+Note on PHP 8.0 or later
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+`DoctrineModule <https://www.doctrine-project.org/projects/doctrine-module/en/current/index.html>`__
+provides an integration with `laminas-cache <https://docs.laminas.dev/laminas-cache/>`__, which
+currently comes with some storage adapters which are not compatible with PHP 8.0 or later. To
+prevent installation of these unused cache adapters, you will need to add the following to your
+``composer.json`` file:
+
+.. code::
```

Comment:
How can we let RST know this is JSON

Reply:
I'd assume that it is `.. code:: json`, but I am not sure.